sn_cli/src/subcommands/wallet.rs

```diff
@@ -271,22 +283,14 @@ pub(super) struct ChunkedFile {
     pub chunks: Vec<(XorName, PathBuf)>,
 }
 
-pub(super) async fn chunk_and_pay_for_storage(
+pub(super) async fn chunk_path(
     client: &Client,
     root_dir: &Path,
     files_path: &Path,
-    verify_store: bool,
-    batch_size: usize,
 ) -> Result<BTreeMap<XorName, ChunkedFile>> {
     trace!("Starting to chunk_and_pay_for_storage");
```

Comment:
Just a small nit, I think we can move this function to `files.rs`? As we don't do any wallet related stuff with it.

Also the log line might have to be updated.